### ISSUE TITLE
Debug adapter process has terminated unexpectedly (read error)

### ISSUE THREAD

**Issue by PrzemyslawKlys** [PowerShell/vscode-powershell#1579]
### System Details

System Details Output
```
& {"### VSCode version: $(code -v)"; "`n### VSCode extensions:`n$(code --list-extensions --show-versions | Out-String)"; "`n### PSES version: $($pseditor.EditorServicesVersion)"; "`n### PowerShell version:`n$($PSVersionTable | Out-String)"}
### VSCode version: 1.28.1 3368db6750222d319c851f6d90eb619d886e08f5 x64

### VSCode extensions:
aaron-bond.better-comments@2.0.2
abusaidm.html-snippets@0.2.1
AlanWalk.markdown-toc@1.5.6
alefragnani.Bookmarks@9.1.0
azemoh.one-monokai@0.3.7
ban.spellright@3.0.4
batisteo.vscode-django@0.17.0
bierner.markdown-preview-github-styles@0.1.3
daiyy.quick-html-previewer@0.3.0
DavidAnson.vscode-markdownlint@0.20.0
DavidWang.ini-for-vscode@0.0.4
dbaeumer.vscode-eslint@1.6.0
donjayamanne.githistory@0.4.3
donjayamanne.jupyter@1.1.4
donjayamanne.python-extension-pack@1.4.0
DotJoshJohnson.xml@2.3.2
eamodio.gitlens@8.5.6
eg2.tslint@1.0.40
emilast.LogFileHighlighter@2.4.2
emmanuelbeziat.vscode-great-icons@2.1.44
Equinusocio.vsc-material-theme@2.4.2
felixfbecker.php-debug@1.12.6
formulahendry.code-runner@0.9.4
gerane.Theme-dimmed-monokai@0.0.2
gharveymn.nightswitch@1.1.0
GitHub.vscode-pull-request-github@0.2.1
GrapeCity.gc-excelviewer@2.1.26
hdg.live-html-previewer@0.3.0
hnw.vscode-auto-open-markdown-preview@0.0.4
HookyQR.beautify@1.4.4
IBM.output-colorizer@0.1.2
ikappas.phpcs@1.0.5
ionutvmi.reg@0.0.2
JerryHong.autofilename@0.1.3
joaompinto.asciidoctor-vscode@0.15.1
johnpapa.winteriscoming@0.8.0
Kelvin.vscode-sshfs@1.10.0
kisstkondoros.vscode-codemetrics@1.17.3
KnisterPeter.vscode-github@0.30.0
magicstack.MagicPython@1.1.0
mariorodeghiero.vue-theme@1.1.1
mechatroner.rainbow-csv@0.7.0
mkloubert.vs-deploy@14.0.0
mkloubert.vscode-remote-workspace@0.34.0
ms-mssql.mssql@1.4.0
ms-python.python@2018.9.0
ms-vscode.csharp@1.16.2
ms-vscode.PowerShell@1.9.0
ms-vsliveshare.vsliveshare@0.3.790
ms-vsts.team@1.142.0
quicktype.quicktype@12.0.46
redhat.java@0.32.0
robertohuertasm.vscode-icons@7.27.0
Shan.code-settings-sync@3.1.2
stkb.rewrap@1.9.1
streetsidesoftware.code-spell-checker@1.6.10
streetsidesoftware.code-spell-checker-polish@1.0.0
swyphcosmo.spellchecker@1.2.13
TomasHubelbauer.zip-file-system@1.0.0
vangware.dark-plus-material@1.4.3
VisualStudioExptTeam.vscodeintellicode@1.0.4
vscjava.vscode-java-debug@0.14.0
vscjava.vscode-java-pack@0.4.0
vscjava.vscode-java-test@0.10.0
vscjava.vscode-maven@0.11.1
WakaTime.vscode-wakatime@1.2.3
wayou.vscode-todo-highlight@1.0.4
whizkydee.material-palenight-theme@1.9.4
wholroyd.jinja@0.0.8
xaver.clang-format@1.6.2
yzhang.markdown-all-in-one@1.6.2
zhuangtongfa.Material-theme@2.17.3
Zignd.html-css-class-completion@1.11.0


### PSES version: 1.9.0.0

### PowerShell version:

Name                           Value
----                           -----
PSVersion                      5.1.17763.1
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.17763.1
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

```

### Issue Description

I am experiencing a problem with... not able to run code

![image](https://user-images.githubusercontent.com/15063294/46903363-0958c000-ced4-11e8-8f33-4bda6f7043a7.png)

Happens randomly, happens more with newest release 1.9.0, haven't seen (read error) in older versions. But it also can be related to WIndows 10 1809 or OneDrive (I do keep my files on one drive - double security - i've many times were able to save uncommited code that I've deleted just by using onedrive).

#### Expected Behaviour

Not crashing.

#### Actual Behaviour

Crashing, usually have to restart PowerShell for it to work again. F5 stops responding. Sometimes starts responding without restart.

### Attached Logs

[1539415847-32d7f941-f01e-4f22-8529-d050803302021539415844201.zip](https://github.com/PowerShell/vscode-powershell/files/2475240/1539415847-32d7f941-f01e-4f22-8529-d050803302021539415844201.zip)

**Comment by rjmholt** [comment]
@PrzemyslawKlys, the logs you've uploaded there only have the vscode log, no EditorServices log.

I'm suspecting that this is a duplicate of https://github.com/PowerShell/vscode-powershell/issues/1441 though.

**Comment by PrzemyslawKlys** [comment]
When I attach zip it's usually what is there with logs. This must mean EditorServices.log was not created :/

**Comment by TheoNordstrom** [comment]
I also experience this error from time to time - it hits randomly, but getting worse after upgrade to v1.10.0 and v1.10.1 this morning. I have attached my EditorServices.log 

[EditorServices.log](https://github.com/PowerShell/vscode-powershell/files/2662829/EditorServices.log)

**Comment by macfarmw** [comment]
I updated to the latest version of the PowerShell extension 1.12.0 released March 29,2019. Since then I have been unable to use the debugger at all. Prior to this release I noticed that a PowerShell command prompt would open when I opened a PowerShell script file. If I closed the command prompt it would prompt me and ask if I wanted to start a new one. None of that is happening now with this new version.
